drakrun/drakrun/draksetup.py

```diff
@@ -83,6 +83,54 @@ def check_root():
         return True
 
 
+@click.command(help='Cleanup the changes made by draksetup')
+def cleanup():
+    if not check_root():
+        return
+
+    install_info = InstallInfo.try_load()
+
+    if install_info is None:
+        logging.error("The cleanup has been performed")
+        return
+
+    backend = get_storage_backend(install_info)
+    vm_ids = get_all_vm_conf()
+
+    for vm_id in vm_ids:
+        delete_vm_conf(vm_id)
+        vm = VirtualMachine(backend, vm_id)
+        vm.destroy()
```

Comment:
If we've stopped drakruns, this shouldn't be necessary

Reply:
Will the same happen with `drakplayground` running?

Reply:
I don't think that cleanup and playground should be running at the same time

Reply:
> I don't think that cleanup and playground should be running at the same time

Yeah I know it shouldn't be. But what if it happens? Shouldn't the cleanup perform necessary measures to ensure that the VM doesn't get corrupted.

Users can make a lot of mistakes, I am just trying to think of robust solutions that might help. If closing the VM when `playground` is running messes up the installation, then I will check if the `playground` is running and issue a warning to user and not run the cleanup